packages/trpc/server/routers/viewer/dsync/teamGroupMapping/_router.tsx

```diff
@@ -0,0 +1,62 @@
+import authedOrgAdminProcedure from "@calcom/trpc/server/procedures/authedProcedure";
+import { router } from "@calcom/trpc/server/trpc";
+
+import { ZCreateInputSchema } from "./create.schema";
+import { ZDeleteInputSchema } from "./delete.schema";
+
+type TeamGroupMappingHandlerCache = {
+  get?: typeof import("./get.handler").getHandler;
+  create?: typeof import("./create.handler").createHandler;
+  delete?: typeof import("./delete.handler").deleteHandler;
+};
+
+const UNSTABLE_HANDLER_CACHE: TeamGroupMappingHandlerCache = {};
+
+export const teamGroupMappingRouter = router({
```

Comment:
Let's follow this boilerplate for trpc routers https://github.com/calcom/cal.com/pull/10849/files